README.md

```diff
@@ -34,12 +34,11 @@ Moreover, the component can shows name initials or simple value as avatar.
 
 ## Installation
 
-Install avatar component using [NPM](https://www.npmjs.com/)::
+Install avatar component using [Yarn](https://yarnpkg.com/)::
 
 ```bash
-$ npm install ngx-avatar --save
+$ yarn add ngx-avatar
```

Comment:
Maybe we should keep both examples, for convenience.

Reply:
done